app/main.py

```diff
@@ -1,6 +1,24 @@
+import os
+import requests
+
+
+URL = "https://api.weatherapi.com/v1/current.json?"
+CITY = "Paris"
+COUNTRY = "France"
+API_KEY = os.getenv("API_KEY")
+
 def get_weather() -> None:
-    # write your code here
-    pass
+
+    response = requests.get(URL + f"q={CITY}&key={API_KEY}")
+    data = response.json()
```

Comment:
Consider adding error handling for JSON parsing. If the response is not in the expected format, this line could raise an exception.

Reply:
Here the JSON is specified by the task condition.